src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -926,9 +926,14 @@ public IPointerTypeSymbol CreatePointerTypeSymbol(ITypeSymbol pointedAtType)
         /// Gets the type within the compilation's assembly and all referenced assemblies (other than
         /// those that can only be referenced via an extern alias) using its canonical CLR metadata name.
         /// </summary>
-        /// <returns>Null if the type can't be found.</returns>
+        /// <returns>Null if the type can't be found, or if multiple assemblies contain a matching type.</returns>
         /// <remarks>
         /// Since VB does not have the concept of extern aliases, it considers all referenced assemblies.
+        ///
+        /// When multiple assemblies contain a matching type, this method returns null and cannot be used to get the types.
```

Comment:
>When multiple assemblies contain a matching type, this method returns null and cannot be used to get the types. [](start = 12, length = 111)

I believe this API also gives preference to types found in the assembly associated with the compilation. Please confirm and, if that is the case, include this information too.